qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -40,19 +46,17 @@ class InstructionToSignals:
 
     The :class:`InstructionsToSignals` class converts a pulse schedule to a list of signals that can
     be given to a model. This conversion is done by calling the :meth:`get_signals` method on a
-    schedule. The converter applies to instances of :class:`~qiskit.pulse.Schedule`. Instances of
-    :class:`~qiskit.pulse.ScheduleBlock` must first be converted to :class:`~qiskit.pulse.Schedule`
-    using the :func:`~qiskit.pulse.transforms.block_to_schedule` function in Qiskit Pulse.
-
-    The converter can be initialized with the optional arguments ``carriers`` and ``channels``. When
-    ``channels`` is given, only the signals specified by name in ``channels`` are returned. The
-    ``carriers`` dictionary specifies the analog carrier frequency of each channel. Here, the keys
-    are the channel name, e.g. ``d12`` for drive channel number ``12``, and the values are the
-    corresponding frequency. If a channel is not present in ``carriers`` it is assumed that the
-    analog carrier frequency is zero.
-
-    See the :meth:`get_signals` method documentation for a detailed description of how pulse
-    schedules are interpreted and translated into :class:`.DiscreteSignal` objects.
+    schedule. The converter applies to instances of :class:`Schedule`. Instances of
```

Comment:
I think something's happened here - this is undoing some changes I had made in a previous PR (that I don't think you are intentionally trying to change).

Reply:
I am so sorry to confuse you. As you say, it is not my intention.  I restored to that of main at .